github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -2087,6 +2087,10 @@ periodics:
             --github-token-file /etc/github/oauth
         cd ${sig_release_dir}
         git add upcoming-changes.md
```

Comment:
Should this check happen before the `git add` ?

Reply:
I don't think that is necessary, we are doing it in the same way in [git-pr.sh](https://github.com/kubevirt/project-infra/blob/main/hack/git-pr.sh#L151)

Reply:
Ok thanks @dhiller